mParticle-Apple-SDK/Include/MPKitProtocol.h

```diff
@@ -144,6 +144,10 @@
                                        callbacks:(MPRoktEventCallback * _Nullable)callbacks
                                     filteredUser:(FilteredMParticleUser * _Nonnull)filteredUser;
 
+- (nonnull MPKitExecStatus *)purchaseFinalized:(nonnull NSString *)placementId
+                                 catalogItemId:(nonnull NSString *)catalogItemId
+                                       success:(NSNumber * _Nullable)success;
```

Comment:
is success actually nullable?  [per the kit code](https://github.com/mparticle-integrations/mparticle-apple-integration-rokt/pull/12/files#diff-09961b2837d7729eae6e2c8c4aec512bda7efbc0901b0b59410bd17d5fcc36ccR265), seems like it's required